riff-cnb-clusterbuildtemplate.yaml

```diff
@@ -22,54 +22,82 @@ spec:
     - name: BUILDER_IMAGE
       description: The builder image provides the buildpacks that will build the IMAGE from source.
       default: projectriff/builder:latest
-    - name: UTIL_IMAGE
-      description: The util image provides pack utilities to analyze and export built images.
-      default: packs/util
     - name: USE_CRED_HELPERS
       description: Use Docker credential helpers for Google's GCR, Amazon's ECR, or Microsoft's ACR.
       default: 'true'
+    - name: USER_ID
+      description: The user ID of the builder image user
+      default: "1000"
+    - name: GROUP_ID
+      description: The group ID of the builder image user
+      default: "1000"
   steps:
     - name: prepare
-      image: ${UTIL_IMAGE}
-      command: ["/lifecycle/knative-helper"]
+      image: ubuntu:18.04
+      command: ["/bin/sh"]
+      args:
+        - "-c"
+        - >
+          chown -R "${USER_ID}:${GROUP_ID}" "/builder/home" &&
```

Comment:
Can we add a comment that explains why those 2 first lines are necessary? Never got to tracking that down

Reply:
I'll add a comment. The packs tasks run as the pack user, and still need to be able to read these files.  This behavior used to be packaged up in the `/lifecycle/knative-helper` which has since been removed.